modules/Bio/EnsEMBL/Compara/RunnableDB/CreateDCJiraTickets.pm

```diff
@@ -0,0 +1,54 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::CreateDCJiraTickets
+
+=head1 DESCRIPTION
+
+A compara runnable to run the create_datacheck_tickets.pl in a pipeline
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::CreateDCJiraTickets;
+
+use warnings;
+use strict;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BaseRunnable');
+
+sub run {
+    my $self = shift;
+
+    my $jira_exe = $self->param_required('create_datacheck_tickets_exe');
+
+    my $command = $jira_exe . ' ' . $self->param_required('output_results') . " --update";
+    $self->warning( "Command: " . $command );
+
+    unless ( $self->param('dry_run') ) {
+        if (defined $ENV{'USERPW'}) {
+            $self->run_command($command, { die_on_failure => 1, });
+        }
+        else {
+            $self->warning( "ENV variable not defined: \$USERPW" );
+        }
```

Comment:
I understand the need of this environment variable, but I'm not sure how I feel about having this password written in a document somewhere in the farm. I'll wait for the other reviewer's opinion.

Reply:
I have a solution to this which I've just successfully tested. I will tidy it up and we can figure out how to proceed with incorporating my changes into yours.

Reply:
Ok, plan is to merge this PR and I will submit a new one to 'overwrite' this part of the code.